A3-Antistasi/flagaction.sqf

```diff
@@ -99,7 +99,7 @@ switch _tipo do
 		fuego addAction ["Rest for 8 Hours", "skiptime.sqf",nil,0,false,true,"","(isPlayer _this) and (_this == stavros)",4];
 		fuego addAction ["Clear Nearby Forest", "clearForest.sqf",nil,0,false,true,"","(isPlayer _this) and (_this == stavros)",4];
 		fuego addAction ["On\Off Lamp", "onOffLamp.sqf",nil,0,false,true,"","(isPlayer _this)",4];
-		fuego addAction ["I hate the fog", "[10,0] remoteExec [""setFog"",2]",nil,0,false,true,"","(_this == stavros)",4];
+		fuego addAction ["Clear Fog", "[10,0] remoteExec [""setFog"",2]",nil,0,false,true,"","(_this == stavros)",4];
```

Comment:
i actually like the light tone of this.
@Barbolani77 ?

Reply:
Well, that's definetly not a lack of english but just a little joke...

Reply:
that's what i thought, @Azylacus can you remove this change ? thx